www/gatsby-node.js

```diff
@@ -35,20 +44,42 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
           reject(result.errors)
         }
 
+        const blogPosts = _.filter(
+          result.data.allMarkdownRemark.edges,
+          edge => {
+            const slug = _.get(edge, `node.fields.slug`)
+            const draft = _.get(edge, `node.frontmatter.draft`)
+            if (!slug) return
+
+            if (_.includes(slug, `/blog/`) && !draft) {
+              return edge
+            }
+          }
+        )
+
+        // Create blog pages.
+        blogPosts.forEach((edge, index) => {
+          const next = index === 0 ? false : blogPosts[index - 1].node
```

Comment:
you can just query for this stuff directly btw :-) `allMarkdownRemark { edges { next { id }, prev { id }, node { id }}}`

Gotta get the GraphQL docs written sigh...